pages/transactional-email/how-to/manage-blocklists.mdx

```diff
@@ -0,0 +1,63 @@
+---
+meta:
+  title: How to manage Transactional Email blocklists
+  description: This page provides instructions how to improve the quality of your email deliveries and maintain your domain's reputation score with TEM blocklists.
+content:
+  h1: How to manage Transactional Email blocklists
+  paragraph: This page provides instructions how to improve the quality of your email deliveries and maintain your domain's reputation score with TEM blocklists.
+tags: blocklist reputation transactional
+dates:
+  validation: 2025-01-20
+  posted: 2025-01-20
+categories:
+  - managed-services
+---
+
+The Transactional Email blocklist feature allows your to manage email addresses that generate errors, or “bounces”. This helps you improve the quality of your email deliveries, maintain your domain's reputation score, and reduce the risk of having your domains blocked by ISPs.
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
```

Comment:
It needs to describe the exact permission list it needs to manage blocklist. As a user, arriving to the full permission page doesn't help me much. Could be frustrating to not find info close to the page documenting the feature.